### ISSUE TITLE
User cannot use keyboard navigation to click on routerlinks in the navbar (Firefox)

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9186]
## Observed behavior
Using keyboard navigation in the appbar/navbar skips over the `routerlinks` (e.g. Channels, Permissions, Facilities, etc. when you are in `Device`)
![2022-03-14 14 29 00](https://user-images.githubusercontent.com/13563002/158266278-3e1d12b5-2c82-4f9e-b137-e06bdecabd2c.gif)

## Errors and logs
None

## Expected behavior
User should be able to navigate through the navigation links in the navbar with the keyboard

## User-facing consequences
Someone using keyboard won't be able to access those router links.

## Steps to reproduce
1. Sign in
2. Use the keyboard to navigate the page after signing in
3. Notice that you'll be unable to access the links in the navbar

## Context
Kolibri 0.15
Firefox

**Comment by marcellamaki** [comment]
Possible Mac settings issue ![tab-click-to-page.gif](https://images.zenhubusercontent.com/6047f6e691103a78d3ad0ace/26be049d-d800-4a61-ac5c-9741f7c8f141)

**Comment by marcellamaki** [comment]
cc @radinamatic -- to double check before closing

**Comment by pcenov** [comment]
Confirmed - with the correct settings enabled the tab navigation works correctly on Safari, Firefox and Chrome on Mac.
